source/test/users/users.spec.ts

```diff
@@ -0,0 +1,60 @@
+import  Request  from 'supertest';
+import Expect from 'chai'
+import {internet} from "faker"
+import Server from '../../server'
+import Users from "../../models/users"
+
+afterEach(() => {
+    Users.deleteMany({name:randomUsername}).exec()
+})
+let randomUsername,
+    randomPassword,
+    presentUsername,
+    presentPassword
+
+describe("Users Test", ()=>{
+    
+    before(()=>{
+        randomUsername=internet.userName();
+        randomPassword = internet.password();
+    })
+
+    it("Add new user - POST/users", async() =>{
```

Comment:
I think this test case is the same with the one next to it? Might be a duplicate.